contracts/modules/SuperMinterV1_1.sol

```diff
@@ -395,6 +420,35 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
         emit Minted(p.edition, p.tier, p.scheduleNum, p.to, l, p.attributionId);
     }
 
+    /**
+     * @inheritdoc ISuperMinterV1_1
+     */
+    function presave(Presave calldata p) public {
+        MintData storage d = _getMintData(LibOps.packId(p.edition, p.tier, p.scheduleNum));
+
+        /* ------------------- CHECKS AND UPDATES ------------------- */
+
+        _requireMintOpen(d);
+
+        if (d.mode != PRESAVE) revert InvalidMode();
+        _verifyAndClaimPresaveSignature(d, p);
+
+        _incrementPresaveMinted(d, p);
+
+        /* ------------------------- MINT --------------------------- */
+
+        unchecked {
+            ISoundEditionV2_1 edition = ISoundEditionV2_1(p.edition);
+
+            uint256 toLength = p.to.length;
+            for (uint256 i; i != toLength; ++i) {
+                edition.mint(p.tier, p.to[i], p.signedQuantity);
+            }
+        }
+
+        emit Presaved(p.edition, p.tier, p.scheduleNum, p.to, p.signedQuantity);
```

Comment:
what's the max size of "to" supported before this out of gas?

does omitting the event have any significant gas benefits?

Reply:
The bulk of gas will be spent on the SSTOREs in the minting. 

My napkin math estimates it at 1000 addresses.

Emitted events do not count towards calldata costs. 30M L2 gas is insignificant compared to calldata costs.